menas/src/main/scala/za/co/absa/enceladus/menas/auth/MenasAuthenticationSuccessHandler.scala

```diff
@@ -48,16 +48,11 @@ class MenasAuthenticationSuccessHandler @Autowired()(jwtFactory: JwtFactory,
     val jwtExpirationTime = DateTime.now(DateTimeZone.forID(timezone)).plus(expiry).toDate
     val cookieLifetime = expiry.getSeconds
 
-    val groups = authentication
-      .getAuthorities.asScala
-      .map(_.getAuthority).asJava
-
     val jwt = jwtFactory
       .jwtBuilder()
       .setSubject(user.getUsername)
       .setExpiration(jwtExpirationTime)
       .claim(CsrfTokenKey, csrfToken)
-      .claim(GroupsKey, groups)
```

Comment:
What was the purpose of including the groups?

Reply:
They were part of the principal and showed up on `/api/user/info`, so I preserved them when switching to JWT. But they were never used anywhere and there might not be a reason for them any time soon if ever.